examples/utils/bundle-size-tests/src/debugAssert.ts

```diff
@@ -8,8 +8,8 @@ import { debugAssert, assert } from "@fluidframework/core-utils/internal";
 // TODO: ideally there would be a unit test which actually checks that this is omitted from production builds.
 // For now it can be manually verified in ../build/debugAssert.js,
 // and any regression breaking it will show up as a bundle size regression.
-debugAssert(() => "This should be removed in production");
-assert(true, "This should be kept 1");
+debugAssert(() => "This should be removed in production 1");
+assert(true, "This should be kept 1", () => "This should be removed in production 2");
```

Comment:
I don't know how our assert tagging works - would we need to make any changes to that infra to accommodate the API changes here?

Reply:
No. The tagging is based on function name and argument index: it leaves all other arguments alone. So the debug message will be left by tagging to be removed by the bundler in production configs.

If there is something likely impacted it would be code which looks at the message from asserts.

Looks like validateAssertionError probably needs and update to support this.

Reply:
Updated validateAssertionError